clients/fides-js/src/components/fides.css

```diff
@@ -1046,13 +1046,20 @@ div#fides-overlay-wrapper .fides-i18n-pseudo-button {
   position: absolute;
   left: 0;
   bottom: 100%;
-  height: 0px;
-  overflow: hidden;
+  visibility: hidden;
   display: flex;
   flex-direction: column;
   gap: 1px;
   max-height: calc(var(--fides-base-font-size) * 7);
-  transition: height 0.5s;
```

Comment:
not sure why the transition never worked, but if we could get it working that would be a bonus. It's a bit jarring when the menu opens without the animation.